snippets/network-flow-matrix.csv

```diff
@@ -0,0 +1,67 @@
+Direction,Protocol,Port,Namespace,Service,Pod,Container,Node Role,Optional
```

Comment:
It says in the text that the table is only for ingress flows, so is the direction column needed if it's all ingress?

Reply:
We might add egress in the future, for now we were thinking it is still a valuable field for a user analyzing the matrix

Reply:
Good point Kerry - but yeah we are hoping to add egress and automate generating the  matrix further. So leaving as it is for now.

Reply:
Might be easier for a user to find two tables (ingress and egress) if you put them under IDs so that they are in the navigation. Just a thought.